cmd/publishing-bot/publisher.go

```diff
@@ -357,6 +357,8 @@ func (p *PublisherMunger) publish(newUpstreamHeads map[string]plumbing.Hash) err
 		if err := os.Chdir(dstDir); err != nil {
 			return err
 		}
+
+		p.plog.Infof("Pushing branches for %s", repoRules.DestinationRepository)
```

Comment:
Without this logs look like:

```
I0309 17:59:15.875382       1 publisher.go:311] Successfully constructed release-1.14
I0309 17:59:15.875490       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token master
I0309 17:59:17.020950       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.11
I0309 17:59:17.361435       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.12
I0309 17:59:17.773017       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.13
I0309 17:59:18.191198       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.14
I0309 17:59:18.516023       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token master
I0309 17:59:21.205627       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.11
I0309 17:59:21.534005       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.12
I0309 17:59:21.854622       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.13
I0309 17:59:22.182615       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.14
I0309 17:59:22.575240       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token master
I0309 17:59:23.009048       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.11
I0309 17:59:23.351475       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.12
I0309 17:59:23.690145       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.13
I0309 17:59:24.072730       1 publisher_logger.go:80] /publish_scripts/push.sh /etc/secret-volume/token release-1.14
----
```

Which makes it hard to understand which repo is being pushed.